hacspec/src/vec_util.rs

```diff
@@ -0,0 +1,79 @@
+#![allow(dead_code)]
+///!
+///! This module holds utility functions for vector manipulation.
+///! This includes sequences, arrays, and polynomials.
+///!
+use crate::prelude::*;
+
+#[inline]
+/// Pad (append) a slice `v` to length `l` with `T::default()`.
+pub(crate) fn pad<T: Numeric + Copy>(v: &[T], l: usize) -> Vec<T> {
```

Comment:
This should be tagged `internal(hacspec)`

Reply:
Indeed. All functions in here should be tagged as internal.